api/src/main/java/org/apache/iceberg/transforms/Bucket.java

```diff
@@ -29,14 +29,17 @@
 import java.nio.charset.StandardCharsets;
 import java.util.Set;
 import java.util.UUID;
-import org.apache.iceberg.expressions.BoundLiteralPredicate;
+import java.util.stream.Collectors;
 import org.apache.iceberg.expressions.BoundPredicate;
-import org.apache.iceberg.expressions.BoundUnaryPredicate;
 import org.apache.iceberg.expressions.Expressions;
 import org.apache.iceberg.expressions.UnboundPredicate;
 import org.apache.iceberg.types.Type;
 import org.apache.iceberg.types.Types;
 
+import static org.apache.iceberg.expressions.Expression.Operation.EQ;
+import static org.apache.iceberg.expressions.Expression.Operation.IN;
+import static org.apache.iceberg.expressions.Expression.Operation.NOT_EQ;
+import static org.apache.iceberg.expressions.Expression.Operation.NOT_IN;
```

Comment:
I think the preferred style is to use `Operation.EQ` inline instead of using static imports.